ethereum/contracts/zksync/Config.sol

```diff
@@ -62,7 +62,7 @@ uint256 constant COMMIT_TIMESTAMP_APPROXIMATION_DELTA = $$(
 );
 
 /// @dev Bit mask to apply for verifier public input before verifying.
-uint256 constant INPUT_MASK = $$(~uint256(0) >> 8);
+uint256 constant INPUT_MASK = $$(~uint256(0) >> 32);
```

Comment:
Why?

Reply:
That’s how public input is constructed/reconstructed in new Boojum and it’s wrapper